<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProfileDialog</class>
 <widget class="QDialog" name="CreateProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Profile</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/MIDAS_Desktop_Icon.png</normaloff>:/icons/MIDAS_Desktop_Icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Profile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="profileComboBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delete this profile</string>
       </property>
       <property name="text">
        <string>Delete Profile</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/delete_small.png</normaloff>:/icons/delete_small.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="4">
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/icons/lock.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="4">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="4">
      <widget class="QLineEdit" name="serverURLEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/icons/earth_network.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Server URL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLineEdit" name="profileNameEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Profile Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/icons/dude3.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QLineEdit" name="rootDirEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="rootDirCheckbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Root Directory</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/icons/gpl_folder_open.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="browseButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QCheckBox" name="anonymousCheckBox">
       <property name="toolTip">
        <string>Anonymous access</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Anonymous</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>profileComboBox</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>profileNameEdit</tabstop>
  <tabstop>anonymousCheckBox</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>serverURLEdit</tabstop>
  <tabstop>rootDirCheckbox</tabstop>
  <tabstop>rootDirEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
  <include location="../MIDASDesktopIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
